samples/bot-commands-menu/nodejs/appManifest/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.17/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.17",
+  "version": "1.0.1",
+  "id": "${{TEAMS_APP_ID}}",
+  "developer": {
+    "name": "Contoso",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Commands Menu",
+    "full": "Commands Menu"
+  },
+  "description": {
+    "short": "Create a commands menu",
+    "full": "This sample add a command menu with a dropdown list of commands for your bot.Users can select a command from the list, which will insert the command string into the message box. Then, they can select Send to execute the command."
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "accentColor": "#60A18E",
+  "bots":[
+    {
+      "botId": "${{AAD_APP_CLIENT_ID}}",
+      "scopes": [
+        "personal",
+        "team"
+      ],
+      "commandLists":[
+        {
+          "scopes":[
+            "team",
```

Comment:
Team scope is not necessary for this sample right?

Reply:
I have removed the team scope as it is not necessary for this sample.